openassessment/xblock/static/js/src/studio/oa_edit_prompts.js

```diff
@@ -21,7 +23,10 @@ OpenAssessment.EditPromptsView = function(element, notifier) {
             removeButtonClass: "openassessment_prompt_remove_button",
             containerItemClass: "openassessment_prompt",
             notifier: notifier,
-            addRemoveEnabled: this.addRemoveEnabled
+            addRemoveEnabled: this.addRemoveEnabled,
+            containerItemsKwargs: {
```

Comment:
Instead of "containerItemsKwargs", name this field "addWysiwyg" and have it pass it the value of "addWysiwyg" from above.